index.bs

```diff
@@ -417,22 +433,17 @@ To <dfn>migrate an animator instance</dfn> from one {{WorkletGlobalScope}} to an
 
         If |definition| does not exist then abort the following steps.
 
-     3. Let |animatorCtor| be the <a>class constructor</a> of |definition|.
+     3. Let |destroyFunction| be the <a>destroy function</a> of |definition|.
 
-     4. Let |prototype| be the result of <a>Get</a>(|animatorCtor|, "prototype").
-
-     5. Let |onDestroyFunction| be the result of <a>Get</a>(|prototype|, "onDestroy").
-
-     6. If the result of <a>IsCallable</a>(|onDestroyFunction|) is false then abort the following
-        steps.
 
-     7. <a>Invoke</a> |onDestroyFunction| with |instance| as the <a>callback this value</a> and let
-        |state| be the result of the invocation.
+     4. <a>Invoke</a> |onDestroyFunction| with |instance| as the <a>callback this value</a> and
+        let |state| be the result of the invocation. If any exception is thrown, then abort the
```

Comment:
Does one need to say 'rethrow the exception' here? Do we want to rethrow it?

Reply:
Yeah. I was not sure what should happen here but let's add rethrow to be consistent.

In general we need a better procedure for when migration fails e.g., when callback 
fails to produce a clonable value. One option is to allow the animation to be aborted perhaps we want to abort it. This is not related to this patch and I plan to re-write this section soon and will address this issue then.